manifests/rhoai/shared/apps/model-catalog/model-catalog-configmap.yaml

```diff
@@ -0,0 +1,319 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
```

Comment:
we'll need some labels here to make sure this is unmanaged / not reconciled by the operator if it's changed manually.
```yaml
  labels:
    opendatahub.io/dashboard: "true"
    opendatahub.io/managed: "false"
```
cc @andrewballantyne

Reply:
As noted in [slack](https://redhat-internal.slack.com/archives/C07C0447EAV/p1740680228381989?thread_ts=1740671586.369199&cid=C07C0447EAV)... we need a change away from 
```
    opendatahub.io/managed: "false"
```

Drop the label entirely. Happy to explain more in slack.

@alexcreasy FYI

Reply:
i think what you are looking for is the annoation not the label if you would like operator not manage your configmap which comes from manifests